ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml

```diff
@@ -198,14 +198,12 @@ NOVA_GID: 998
 barbican_encryption: "True"
 
 
+## TASKFLOW PARAM
```

Comment:
Why is this in caps ?

Reply:
it's a comment only

Reply:
see below lines

Reply:
@dhiraj-trilio these are the vars kind of placeholders.

Reply:
can we keep in lower case just like other variables in this file eg. barbican_encryption, wlm_log_dir ? ideally we place all variables in conf file in lowercase letter only. This file is mixed of upper and lower case variables. can we keep all these in lowercase ?

Reply:
@dhiraj-trilio yes will do it in 5.0 SP1